modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.jsx

```diff
@@ -603,7 +603,7 @@ const RequiredInformation = ({
 								</div>
 							) : (
 								<div className="mx-6">
-									{!isOemOrEnterprise && (
+									{!isComplimentaryKey || !isOemOrEnterprise && (
```

Comment:
should this be an `&&` rather than an `||`?

Reply:
@kylembischof , you're right it, this breaks for oem now. I'll change it in the cherry-pick

Reply:
actually I was looking at the wrong thing. I don't even know how to get to this part but your logic should be correct

Reply:
@kylembischof I changed it to `&&`. I will continue with the tests.